conn/physic/units.go

```diff
@@ -70,7 +70,7 @@ func (a *Angle) Set(s string) error {
 		if e, ok := err.(*parseError); ok {
 			switch e.err {
 			case errNotANumber:
-				if found, _ := containsUnitString(s[n:], "rad", "deg", "°"); found != "" {
+				if found, _ := containsUnitString(s[n:], "Rad", "rad", "Deg", "deg", "°"); found != "" {
```

Comment:
I just noticed there is no test for correctness of containsUnitString().
As it is it will convert both the potential unit string and the valid strings args to lower case for the comparison.